src/content/docs/tips/tailwind-v4.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Tailwind v4
+description: Quick set up guide for Tailwind v4
+---
+import { Tabs, TabItem, LinkCard } from "@astrojs/starlight/components";
+
+How to use Tailwind's v4 Vite plugin instead of Astro's Tailwind integration.
+
+Be sure to check out the [Tailwind v4 Blogpost](https://tailwindcss.com/blog/tailwindcss-v4-alpha#using-vite) for details on using v4.
+
+## Install
+
+<Tabs>
+    <TabItem label="npm">
+        ```sh
+        npm install --save-dev @tailwindcss/vite
+        ```
+    </TabItem>
+    <TabItem label="pnpm">
+        ```sh
+        pnpm add -D @tailwindcss/vite
+        ```
+    </TabItem>
+    <TabItem label="yarn">
+        ```sh
+        yarn add --dev @tailwindcss/vite
+        ```
+    </TabItem>
+</Tabs>
+
+## Configure
+
+```ts title="astro.config.mjs"
```

Comment:
```suggestion
```ts title="astro.config.mjs" {tailwindcss()} {7-10}
```

Reply:
I was just gonna suggest the same 🚀